src/main/java/org/cryptomator/cryptofs/health/dirid/MissingDirFile.java

```diff
@@ -0,0 +1,45 @@
+package org.cryptomator.cryptofs.health.dirid;
+
+import org.cryptomator.cryptofs.VaultConfig;
+import org.cryptomator.cryptofs.health.api.CommonDetailKeys;
+import org.cryptomator.cryptofs.health.api.DiagnosticResult;
+import org.cryptomator.cryptolib.api.Cryptor;
+import org.cryptomator.cryptolib.api.Masterkey;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+/**
+ *  A c9r directory without a dirId file.
+ */
+public class MissingDirFile implements DiagnosticResult {
```

Comment:
Same. It is not a file containing a missing dir, but it is the dir id file that is missing.

Reply:
Changed in [82775e7](https://github.com/cryptomator/cryptofs/pull/138/commits/82775e7e4fce91350a0b461bf4512126a2b88b42)